contents/tutorindigo.rst

```diff
@@ -0,0 +1,245 @@
+- Similar to above steps, you can update all images including favicon.ico
+
+.. image:: /academy/static/images/tutorindigo/logo_favicon.png
+  :width: 800
+  :alt: Logo and Favicon Example Image
+
+Font Modifications
+------------------
+
+There are two ways to update font family of LMS. One is to use your own font files and second is to use google fonts url. Follow the below steps to update font of LMS:
+
+- Clone the Tutor Indigo by running the command ``git clone https://github.com/overhangio/tutor-indigo.git``
+- Download your font files and move them to ``tutorindigo/templates/indigo/lms/static/fonts`` folder. For example, you want to change LMS font family to "Crimson". Download your files and move to the ``tutorindigo/templates/indigo/lms/static/fonts`` folder
+- Now, setup their path in ``tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_fonts.scss`` file. If the file is not present, then create it and add the font files path to it. In case of Crimson font, you need to add this:
+
+.. code-block:: sass
+
+    @font-face {
+        font-family: 'Crimson';
+        src: url('#{$static-path}/fonts/CrimsonText-Regular.ttf') format('truetype');
+        font-weight: 400;
+        font-style: normal;
+    }
+
+    @font-face {
+        font-family: 'Crimson';
+        src: url('#{$static-path}/fonts/CrimsonText-Italic.ttf') format('truetype');
+        font-weight: 400;
+        font-style: italic;
+    }
+
+    @font-face {
+        font-family: 'Crimson';
+        src: url('#{$static-path}/fonts/CrimsonText-SemiBold.ttf') format('truetype');
+        font-weight: 600;
+        font-style: normal;
+    }
+
+    @font-face {
+        font-family: 'Crimson';
+        src: url('#{$static-path}/fonts/CrimsonText-SemiBoldItalic.ttf') format('truetype');
+        font-weight: 600;
+        font-style: italic;
+    }
+
+    @font-face {
+        font-family: 'Crimson';
+        src: url('#{$static-path}/fonts/CrimsonText-Bold.ttf') format('truetype');
+        font-weight: 700;
+        font-style: normal;
+    }
+
+    @font-face {
+        font-family: 'Crimson';
+        src: url('#{$static-path}/fonts/CrimsonText-BoldItalic.ttf') format('truetype');
+        font-weight: 700;
+        font-style: italic;
+    }
+
+Here, you have to ensure that ``font-weight`` and ``font-style`` should match with the respective files. Like, ``fonts/CrimsonText-Bold.ttf`` has font-weight 700 and font-style normal.
+
+- If you don't want to add font files and use Google font URL. Then, add the below url in the ``tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_fonts.scss`` file (for `Crimson font family <https://fonts.google.com/specimen/Crimson+Text>`_)::
```

Comment:
This is confusing. We are describing two different methods of doing the same thing. I suggest we remove this section. People who want to load fonts from Google will guess how to achieve that.